src/audio/wav64.c

```diff
@@ -255,19 +273,12 @@ static void waveform_vadpcm_read(void *ctx, samplebuffer_t *sbuf, int wpos, int
 void wav64_open(wav64_t *wav, const char *fn) {
 	memset(wav, 0, sizeof(*wav));
 
-	// Currently, we only support streaming WAVs from DFS (ROMs). Any other
-	// filesystem is unsupported.
-	// For backward compatibility, we also silently accept a non-prefixed path.
-	if (strstr(fn, ":/")) {
-		assertf(strncmp(fn, "rom:/", 5) == 0, "Cannot open %s: wav64 only supports files in ROM (rom:/)", fn);
-		fn += 5;
-	}
-
-	int fh = dfs_open(fn);
-	assertf(fh >= 0, "error opening file %s: %s (%d)\n", fn, dfs_strerror(fh), fh);
-
+	// Open the input file.
+	int fptr = 0;
+	fptr = open(fn, O_RDONLY);
+	assertf(fptr >= 0, "error opening file %s: FILE == NULL\n", fn);
```

Comment:
There's a function called `must_open()` in `asset.c`. You can declare it in `asset_internal.h` and call it from here. It is a wrapper to `open()` that provides nice error messages in case of common programming mistakes.